Disables opening of ldaprc file in current directory.

Resolves: #38402
Upstream: ITS #1131
Author: Henning Schmiedehausen <hps@intermeta.de>

--- openldap-2.4.24.orig/libraries/libldap/init.c
+++ openldap-2.4.24/libraries/libldap/init.c
@@ -346,9 +346,6 @@ static void openldap_ldap_init_w_usercon
 	if(path != NULL) {
 		LDAP_FREE(path);
 	}
-
-	/* try file */
-	openldap_ldap_init_w_conf(file, 1);
 }
 
 static void openldap_ldap_init_w_env(
